Gamma_Analysis.py

```diff
@@ -0,0 +1,373 @@
+
+    # Returning results
+
+    results = [net_area, uncertainty]
+
+    return results
+
+
+def background_subtract(Meas_Area, Back_Area, Meas_Time, Back_Time):
+    """
+    Background_Subtract will subtract a measured Background peak net area from
+    a sample peak net area. The background peak is converted to the same time
+    scale as the measurement and the subtraction is performed. All inputs are
+    scalar numbers, where Meas_Area and Back_Area represent the net area of
+    a sample net area and background net area respectively. Meas_Time and
+    Back_Time are the livetimes of the measurement and background respectively.
+    """
+    Time_Ratio = Meas_Time/Back_Time
+    Back_to_Meas = Back_Area[0]*Time_Ratio
+    Meas_Sub_Back = Meas_Area[0] - Back_to_Meas
+
+    Meas_Uncertainty = Meas_Area[1]
+    Back_Uncertainty = Back_Area[1]*Time_Ratio
+    Meas_Sub_Back_Uncertainty = (Meas_Uncertainty+Back_Uncertainty)**0.5
+
+    Sub_Peak = [Meas_Sub_Back, Meas_Sub_Back_Uncertainty]
+    return Sub_Peak
+
+
+def make_table(Isotope_List, sample_info, sample_names, dates):
+    data = {}
+
+    for i in range(len(sample_names)):
+        data[sample_names[i]] = sample_info[i]
+
+    Isotope_Act_Unc = []
+    for i in range(len(Isotope_List)):
+        Isotope_Act_Unc.append(str(Isotope_List[i].Symbol) + '-' +
+                               str(Isotope_List[i].Mass_number) +
+                               ' Act' + '[Bq]')
+        Isotope_Act_Unc.append(str(Isotope_List[i].Symbol) + '-' +
+                               str(Isotope_List[i].Mass_number) +
+                               ' Unc' + '[Bq]')
+
+    frame = pd.DataFrame(data, index=Isotope_Act_Unc)
+    frame = frame.T
+    frame.index.name = 'Sample Type'
+
+    #Adding Date Measured and Sample Weight Columns
+    df = pd.read_csv('RadWatch_Samples.csv')
+    frame['Date Measured'] = dates
+    frame['Sample Weight (g)'] = pd.Series.tolist(df.ix[:,2])
+
+    #Reindexing columns to place 'Date Measured' and 'Sample Weight' first.
+    colnames = frame.columns.tolist()
+    colnames = colnames[-2:] + colnames[:-2]
+    frame = frame[colnames]
+    frame.to_csv('Sampling_Table.csv')
+
+    return frame
+
+
+def main():
+    Background = SPEFile.SPEFile("USS_Independence_Background.Spe")
+    Background.read()
+    Reference = SPEFile.SPEFile("UCB018_Soil_Sample010_2.Spe")
+    Reference.read()
+    Sample_Comparison = ref.Soil_Reference
+    dir_path = os.getcwd()
+    Sample_Measurements = []
+    SAMPLE_NAMES = []
+    Measurement_Dates = []
+    Sample_Data = []
+    Error_Spectrum = []
+
+    for file in os.listdir(dir_path):
+        if file.endswith(".Spe"):
+            if file == "USS_Independence_Background.Spe":
+                pass
+            else:
+                Sample_Measurements.append(file)
+                Name = os.path.splitext(file)[0].replace("_", " ")
+                SAMPLE_NAMES.append(Name)
+
+    for SAMPLE in Sample_Measurements:
```

Comment:
Write a function that makes measurements on a single file. Basically all the code below here will go in that function, and then you'll just call that function here. 
You'll want to think about how you organize the data that comes back from the function.